.github/ISSUE_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+## Description
+
+Please include a summary of the changes and the related issue. List any dependencies that are required for this change.
+
+Fixes #(number of issue in github)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
```

Comment:
IMO we should remove this option. If a documentation update is necessary, it should already be part of the PR, this is why the user documentation is part of the project itself.